py3status/modules/volume_status.py

```diff
@@ -155,10 +162,10 @@ def get_volume(self):
         return perc, muted
 
     def volume_up(self, delta):
-        self.run_cmd(self.cmd + ["-i", str(delta)])
+        self.run_cmd(self.cmd + ["-i", delta])
```

Comment:
In general just try to fix a specific problem.  All changes risk introducing bugs.

This change 100% introduces one
```
>>> from subprocess import call
>>> call(['echo', '1'])
1
0
>>> call(['echo', 1])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/subprocess.py", line 168, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib64/python2.7/subprocess.py", line 390, in __init__
    errread, errwrite)
  File "/usr/lib64/python2.7/subprocess.py", line 1024, in _execute_child
    raise child_exception
TypeError: execv() arg 2 must contain only strings
```

Every change in a PR has an overhead and reduces the chances of the PR getting merged.  I know you are keen, and this is a good thing but it helps to be discipline even if the reasoning is not apparent initially.

There is a lot of subtle things going on :)

Reply:
I changed `volume_delta` from integer to string once in `post_config_hook` so we can do `call(['echo', delta])` here. Is that not correct? I tested this too.

Reply:
Thx